ui/src/components/AppDetailsToolbar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { Toolbar, ToolbarGroup, ToolbarItem, Breadcrumb, BreadcrumbItem, Button } from '@patternfly/react-core';
+import './AppDetailedToolbar.css';
+
+export class AppDetailsToolbar extends React.Component {
+  onDisableApp = () => {
+    console.log('Disable button clicked');
+  }
+
+  onHomeClick = () => {
+    this.props.history.push('/');
+  }
+
+  onSaveApp = () => {
+    console.log('Save button clicked');
+  }
+
+  render () {
+    return (
+      <Toolbar className='details-toolbar'>
+        <ToolbarGroup>
+          <ToolbarItem>
+            <Breadcrumb>
```

Comment:
Breadcrumb still looks very small - can it be styled to increase the font a little?